controller/src/main/java/org/jboss/as/controller/remote/TransactionalProtocolOperationHandler.java

```diff
@@ -69,7 +69,7 @@
  * @author <a href="mailto:darran.lofthouse@jboss.com">Darran Lofthouse</a>
  */
 public class TransactionalProtocolOperationHandler implements ManagementRequestHandlerFactory {
-    private static final Set<String> PREPARED_RESPONSE_OPERATIONS = Set.of(RELOAD);
+    private static final Set<String> PREPARED_RESPONSE_OPERATIONS = ModelDescriptionConstants.RELOAD_OPERATIONS;
```

Comment:
Is this change necessary? I would say TransactionalProtocolOperationHandler is only used in domain mode and this RFE is not for Domain Mode.